sway/tree/layout.c

```diff
@@ -557,6 +572,9 @@ void arrange_windows(struct sway_container *container,
 		return;
 	case C_WORKSPACE:
 		{
+			if (container->sway_workspace->fullscreen) {
+				return;
```

Comment:
We may still want to set the workspace w/h/x/y (which is done after the switch), not sure what the ramifications of not doing that will be

Reply:
bump

Reply:
Resolved in 7e38cc2.